Pear/Controller/LoginViewController.swift

```diff
@@ -111,16 +111,31 @@ class LoginViewController: UIViewController {
                     DispatchQueue.main.async {
                         UserDefaults.standard.set(userSession.accessToken, forKey: Constants.UserDefaults.accessToken)
                         
-                        let onboardingVC = OnboardingPageViewController(transitionStyle: .scroll, navigationOrientation: .horizontal)
-                        
-                        self.navigationController?.pushViewController(onboardingVC, animated: true)
+                        self.getUser()
                     }
                 case .failure(let error):
                     print(error.localizedDescription)
                 }
             }
         }
     }
+    
+    private func getUser() {
+        NetworkManager.getMe { [weak self] result in
```

Comment:
`getUser` or `getCurrentUsed`seems like a better name. `getMe` feels informal. Maybe not.

Reply:
yeah getMe is wacky